aten/src/ATen/native/GridSampler.cpp

```diff
@@ -538,7 +538,7 @@ Tensor _grid_sampler_2d_cpu_quantized(
             res += within_bounds_2d(iy_se, ix_se, inp_H, inp_W)
                 ? inp_ptr_NC[iy_se * inp_sH + ix_se * inp_sW] * se
                 : zero_point * se;
-            *out_ptr_NCHW = std::round(res);
+            *out_ptr_NCHW = std::nearbyint(res);
```

Comment:
This rounding change makes the activation quantization round to the nearest even which is usually the rounding mode used for quantized convolution in frameworks such as PyTorch and TensorRT.

Reply:
Does this have to be a separate PR?

Reply:
Is this related to the consistency issue for the nearest rounding mode? If not it might be best to split it into a separate PR

Reply:
This issue is related to quantization activation value rounding, instead of index rounding. So it's a different issue. I will make it separate from this PR.

Reply:
Removed this change from this PR.